pkg/handlers/deploy.go

```diff
@@ -140,5 +144,59 @@ func (h *Handler) DeployAppVersion(w http.ResponseWriter, r *http.Request) {
 
 	deployAppVersionResponse.Success = true
 
-	JSON(w, 204, deployAppVersionResponse)
+	JSON(w, http.StatusOK, deployAppVersionResponse)
+}
+
+type AdminConsoleUpgradeError struct {
+	IsCritical bool
+	Message    string
+}
+
+func (e AdminConsoleUpgradeError) Error() string {
+	return e.Message
+}
+
+func getTargetKotsVersion(kotsKinds *kotsutil.KotsKinds, latestVersion string) (string, error) {
```

Comment:
can we rename this function? since we already have both `targetKotsVersion` and `minKotsVersion`, i had to give this another look to understand what it actually does. (thinking of other names now)

Reply:
yeah